staubli_tx2_90_support/package.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>staubli_tx2_90_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Staubli TX2-90 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Staubli TX2-90 manipulators. This includes the base model,
+      as well as models with extended arm and forearm links (L and XL).
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>TX2-90 - Standard Arm - Nominal Speed</li>
+      <li>TX2-90L - Long Arm - Nominal Speed</li>
+      <li>TX2-90XL - Extra Long Arm - Nominal Speed</li>
+    </ul>
+    <p>
+      Joint limits were taken from broschures, torque limits, and maximum joint velocities
+      are based on the
+      information in the <em>Staubli Robotics - TX series 90 family -
+      Instruction Manual</em> version <em>D28082904C – 01/12/2014</em> and might have changed.
+      All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise (ie: no support for high speed joints, extended /
+      limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+    <p>
+      <b>NB (1)</b>: Masses, center of mass and moments of inertia were copied from TX90 package. There they
+      were calculated using Solidworks and the official Staubli CAD models, and may not be accurate.
```

Comment:
Were these copied from other models (could be that you already answered this, I don't remember)?

The values seem to be duplicated between all three variants. Seeing as the L and XL have a longer reach, I would expect them to be heavier.

I'm wondering whether we shouldn't just remove the `inertial` elements if we don't know any of the proper values and we don't have appropriate estimates.

Reply:
I'm fine with removing them, will do that on Tuesday